### PR TITLE
fixed the error when trying to perform get on channels

### PR DESCRIPTION
**Pull request by KamilPiekutowski**
now it states the get cannot be used on channels

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ at anytime.
 ### Security
   *
   *
+### Fixed
+  * fixed `get` on channel now it returns error that informs of `get` not supporting channels
+  * the message that is displayed can be more descriptive, and it is open for discussion
```
Comment (hackrush01) [inline CHANGELOG.md]: This line can be removed from the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ at anytime.
 ### Security
   *
   *
+### Fixed
```
Comment (hackrush01) [inline CHANGELOG.md]: We already have `Fixed` heading so this should be removed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,9 @@ at anytime.
...
+  * fixed `get` on channel now it returns error that informs of `get` not supporting channels
```
Comment (hackrush01) [inline CHANGELOG.md]: This line should be put in the already existing 'Fixed' section below.

**Comment by hackrush01** [comment]
Some minor changes, otherwise LGTM.

**Comment by KamilPiekutowski** [comment]
@hackrush01 I edited the CHANGELO.md and squashed the commits. Please let me know if that is all you need.

**Comment by lyoshenka** [comment]
merged

**Comment by KamilPiekutowski** [comment]
@lyoshenka  So this one is approved?

**Comment by lyoshenka** [comment]
yep. i cleaned it up a bit and merged it in https://github.com/lbryio/lbry/commit/bb2efdcc7c38a26cd219e8866000e4857b2bffc1

**Comment by KamilPiekutowski** [comment]
@lyoshenka  What should I do with closed unmerged commit? Just delete branch?

**Comment by lyoshenka** [comment]
yep

**Pull request closed without merging**